src/plasma/plasma.h

```diff
@@ -222,6 +222,11 @@ typedef struct {
   object_table_entry *objects;
 } plasma_store_info;
 
+typedef struct {
+  object_id obj_id;
+  int64_t data_size;
+} objectid_notification;
```

Comment:
can we rename this to object_id_notification to be consistent with the rest of the code?